wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/WorkspaceActivityChecker.java

```diff
@@ -153,6 +160,29 @@ private void checkActivityRecordValidity() throws ServerException {
     }
   }
 
+  /**
+   * Makes sure that any activity records are rectified if they do not reflect the true state of the
+   * workspace anymore.
+   *
+   * @throws ServerException or error
```

Comment:
`or error` - it's like `on any error`?

Reply:
well, the description is as useless as the need to declare it in the first place ;) Since `ServerException` is a blanket "something bad happened" and is generally not handled anywhere (or it is just logged and ignored), it makes little sense to be specific it.